roles/system-base/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 # Minimum version of the kernel package that should be installed by this role
-kernel_min: "5.10"
\ No newline at end of file
+kernel_min: "5.10"
+yq_version: "4.16.1"
```

Comment:
I wouldn't bother making this a variable, since we're not likely to ever upgrade it separately. If you want, you could always download the latest yq version with something fancy like:


```
  - name: register latest yq version
    uri:
      url: https://api.github.com/repos/mikefarah/yq/releases/latest
      return_content: true
    register: yq_release

  - name: "installing yq {{ yq_release.json.tag_name }}"
    loop: "{{ yq_release.json.assets }}"
    when: "'yq_linux_amd64' in item.name"
    get_url:
      url: "{{ item.browser_download_url }}"
      dest: /usr/bin/yq
      mode: 0755
```

(I'll leave it up to you if you want to make it autodetect the architecture too. Idk if you can do jinja2 in a `when` block)
(also, I'm saving this snippet for later, since I'm sure it'll be useful in the future...)